apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -102,6 +106,20 @@
       this.initService();
     },
 
+    register(appInstanceId) {
+      var mobileMessageClient = MozMobileMessageClient.forApp(appInstanceId);
+
+      MESSAGE_EVENTS.forEach((event) => {
+        mobileMessageClient.on(event, (data) => {
+          // Workaround to avoid the duplicated events by inserting
+          // appInstanceId into event name in bridge mixin.
+          data.appInstanceId = appInstanceId;
+          this.broadcast(event, data);
+        });
+      });
+      return;
+    },
```

Comment:
In https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/services/README.md#conversationservice we did want to send events only when receiving a message, and when changing an existing message.

That said I understand why you want to change this, but then we need to change the README as well, and maybe think this through. Maybe we don't want message-sending/message-received but message-created ?